src/Stripe.Tests/transfers/test_data/stripe_transfer_create_options.cs

```diff
@@ -18,7 +18,8 @@ public static StripeTransferCreateOptions Valid(string recipient = "self")
                 {
                     { "A", "Value-A" },
                     { "B", "Value-B" }
-                }
+                },
+                ApplicationFee = 10
```

Comment:
I'm not really familiar with the testing framework you're using, but I'll take a stab at it.